litestar/routes/base.py

```diff
@@ -123,23 +118,6 @@ async def handle(self, scope: Scope, receive: Receive, send: Send) -> None:
         """
         raise NotImplementedError("Route subclasses must implement handle which serves as the ASGI app entry point")
 
-    def create_handler_kwargs_model(self, route_handler: BaseRouteHandler) -> KwargsModel:
-        """Create a `KwargsModel` for a given route handler."""
-
-        path_parameters = set()
-        for param in self.path_parameters:
-            if param.name in path_parameters:
-                raise ImproperlyConfiguredException(f"Duplicate parameter '{param.name}' detected in '{self.path}'.")
-            path_parameters.add(param.name)
-
```

Comment:
No need to do this every time we create the kwargs model; For routes with multiple handlers, we'd do this multiple times